AGENTS.md

```
@@ -15,51 +15,134 @@ oc-mirror reads an `ImageSetConfiguration` YAML file and mirrors container image
 
 ### Version structure
-- **v2** (Current) : code lives in the root directory - **THIS IS WHAT YOU SHOULD WORK ON**
-- **v1** (Deprecated) : code lives under the v1/ folder - **DO NOT MODIFY v1 CODE**
+- **v2** (Current) : code lives in the root directory — **THIS IS WHAT YOU SHOULD WORK ON**
+- **v1** (Deprecated) : code lives under the `v1/` folder — **DO NOT MODIFY v1 CODE**
 
-## Key Architecture components
+## Project structure
 
-The `oc-mirror` project relies heavily on the [container-libs](https://github.com/containers/container-libs) library for low-level container image operations.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)